Vector/Assets/en.lproj/Vector.strings

```diff
@@ -51,7 +51,7 @@
 "auth_register" = "Register";
 "auth_send_reset_email" = "Send Reset Email";
 "auth_return_to_login" = "Return to login screen";
-"auth_user_id_placeholder" = "Email or user name";
+"auth_user_id_placeholder" = "Email, user name or phone number";
```

Comment:
user name could appear at first as this will be the 1st thing asked in the 1st registration page.